quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.upgrad.quora.api.controller;
+
+import com.upgrad.quora.api.model.SignupUserResponse;
+import com.upgrad.quora.api.model.SignupUserRequest;
+
+import com.upgrad.quora.service.Entity.UserEntity;
+import com.upgrad.quora.service.business.UserBusinessService;
+import com.upgrad.quora.service.exception.SignUpRestrictedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/")
+public class UserController {
+
+    @Autowired
+    private UserBusinessService userBusinessService;
+
+
+    //The userSignup()  method is of request post type , path is "/user/signup" the method set all the attribute of user in userEntity and generate a Response model with HTTP status indicating successfully registered
```

Comment:
please change this comment to javadoc style comment

Reply:
Done the changes